Modules/QuestieAnnounce.lua

```diff
@@ -86,3 +86,43 @@ function QuestieAnnounce:ItemLooted(text, notPlayerName, _, _, playerName)
     end
 end
 
+---@return table<string, number, string>
+local function _GetQuestInfos(questId)
```

Comment:
You should define these functions at the top of the file first so they are defined before they are used.

At the top you can simply do:

```lua
local _GetQuestInfos
```

and then here it will become:

```suggestion
_GetQuestInfos = function(questId)
```